packages/apps/plugins/plugin-markdown/src/components/EditorMain.tsx

```diff
@@ -103,15 +95,13 @@ const EditorMain = ({ model, comments, toolbar, extensions: _extensions, ...prop
                 textBlockWidth,
                 editorFillLayoutRoot,
                 'md:border-is md:border-ie separator-separator focus-visible:ring-inset',
+                !toolbar && 'border-bs separator-separator',
               ),
               'data-testid': 'composer.markdownRoot',
             } as HTMLAttributes<HTMLDivElement>,
-            editor: {
-              className: mx(editorFillLayoutEditor, !toolbar && 'border-bs'),
-            },
             content: {
-              // after:block after:is-px after:bs-px after:overflow-anchor after:-mbs-px
-              className: mx(editorHalfViewportOverscrollContent, '!p-2 sm:!p-6 md:!p-8'),
+              // TODO(burdon): Override (!) required since base theme sets padding and scrollPastEnd sets bottom.
+              className: mx('!px-2 sm:!px-6 md:!px-8 !pt-2 sm:!pt-6 md:!pt-8'),
```

Comment:
```suggestion
              className: mx('!pli-2 sm:!pli-6 md:!pli-8 !pt-2 sm:!pbs-6 md:!pbs-8'),
```
Please always use logical properties

Reply:
do you have a cheatsheet since the TW docs don't explain these as far as i could find:
https://tailwindcss.com

Reply:
Yes for sure: https://stevecochrane.github.io/tailwindcss-logical/